tests/escape.earth

```diff
@@ -1,3 +1,4 @@
+VERSION 0.6
```

Comment:
So if I'm understanding correctly, we already tested escaping + but because we default to VERSION 0.5 it wasn't caught? So just bumping this version gives us test coverage for this?

Reply:
that's correct, `0.6` enables `--use-copy-include-patterns` by default. prior to this https://github.com/earthly/earthly/blob/main/tests/copy-verbose.earth was the only test we had providing us coverage (which wasn't ample)